### ISSUE TITLE
Imba Rebrand Proposal

### ISSUE THREAD

**Issue by iamtirado** [somebee/imba#140]
# Rebranding Process for Imba
The purpose of this is rebrand is to visually establish Imba as a reputable contender to the major branded web frameworks and languages.

The following steps will hopefully make the process orderly and effective:

## Step 1:
The process will begin by identifying 3-5 major characteristics for the Imba language and community.
- [X] 1.1 Comment below with 5 characteristics that exemplify imba and the community to you, and rank them in order of 1-5

### Result
Here is a tally of the votes of why we love Imba.
```
Fast Performance: 33 votes
Beautiful Syntax: 29 Votes
Capability/Functionality: 21 Votes
Fun to Use: 11 votes
Rebel: 4 votes
```
I simply weighed the votes as follows: 
>1. = 5votes
>2. = 4votes
>3. = 3votes
>4. = 2 votes
>5. = 1 votes

I then, combined the votes for the words that were similar as follows
>Fast Performance: Fast(29) + Responsive(4) = 33
Beautiful Syntax: elegant(9) + clean(5) + simple(7) + easy(3) + comfortable(1) + readable(4) = 29
Capability/Functionality: smart(9) + innovative(5) + expressive(5) + intuitive(2) = 21
Fun to use: funny(2) + playful(5) + fun(3) + quirky(1) = 11
Rebel: rebel(1) + antimainstream(3) + radical = 4

### Conclusion
Developers are drawn to Imba primarily because of its unmatched performance.
Secondly, by its simple, readable syntax.
Thirdly, by its near-limitless functionality.
And lastly, perhaps because the vDOM has such a strong community behind it, you will find that early Imba adopters must have a bit of a rebellious spirit to challenge it.

### Identity of Imba
Since the votes add up to 98, we can treat the votes as a percentage as you think of future contributions to the brand.
>Imba's identity lies 33% on Speed, 29% on Simplicity, 21% on Functionality, 11% fun and 4% Rebellious. 

We can attribute the other 2% to other single votes that didn't get clumped in like fresh or future.

---

## Step 2 - Current Task 🏁 
In this step we're going to implement the Identity mentioned above into different branding elements starting with a tagline.

Current Task 🏁
- [ ] 2.1 In 3~7 words, write a tagline for Imba that embodies Imba's Identity. The less words, the better.
Make each submission in a separate comment, then vote on other submissions with thumbs up 👍 or thumbs down.👎

Future Tasks
- [ ] 2.2 Submit Black & white Logos that embody the identity of Imba
- [ ] 2.3 Submit Color Schemes that embody the identity of Imba, with moodboard or mockups if possible.

### Code of Conduct
> Any creative process will seem subjective to personal opinion to most people, and opinions can be endless. 
- Keep your input short, constructive, and focused around the identity of the brand, rather than personal identity. 
- Address comments to the task at hand, and use reactions to provide feedback.
- For back and forth discussion, discuss in gitter to keep the thread clean.

Special attention to @somebee @madmaniak @altteo @foxbunny @allain @athif23 @perborgen
And anyone else who'd like to contribute.

**Comment by foxbunny** [comment]
1. fast
2. smart
3. simple
4. funny
5. quirky

**Comment by madmaniak** [comment]
Step 1:
1. Smart
2. Readable
3. Fast
4. Innovative
5. Superior

**Comment by somebee** [comment]
1. fast
2. elegant
3. fun
4. playful
5. contrarian / rebel

**Comment by foxbunny** [comment]
![image](https://user-images.githubusercontent.com/65413/48871653-ea5c2080-ede5-11e8-8732-b850f3f12ee7.png)

**Comment by madmaniak** [comment]
+1 for rebel

**Comment by iamtirado** [comment]
1. Elegant
2. Simple
3. Fast
4. The Best
5. Fresh/New

**Comment by allain** [comment]
1. Expressive
2. Fast
3. Radical
4. Playful
5. Innovative

**Comment by athif23** [comment]
1. Fast
2. Responsive
3. Anti-Mainstream
4. Future
5. Playful

**Comment by altteo** [comment]
1. Clean
2. Fast
3. Easy
4. Joyfull
5. Comfortable

**Comment by codeluggage** [comment]
1. Elegant
2. Simple
3. Clean
4. Fast
5. Rebel

**Comment by iamtirado** [comment]
Powering the Fastest Web apps

**Comment by iamtirado** [comment]
2.1 – building the fastest web apps like child's play

**Comment by athif23** [comment]
I have two,
2.1 - Simply and smartly building the fastest web apps
or
2.1 - Create the fastest web app with simplicity

**Comment by iamtirado** [comment]
2.1 – Easy language for disruptively fast web apps!
(Easy + Fast + Rebelious)

**Comment by iamtirado** [comment]
> I have two,
> 2.1 - Simply and smartly building the fastest web apps
> or
> 2.1 - Create the fastest web app with simplicity

Can you separate for independent voting?

**Comment by madmaniak** [comment]
I'd like us to remember about yesterday's @foxbunny joke, which actually sounds good:

2.1 - Macho dude with arms like a tag

**Comment by foxbunny** [comment]
@madmaniak In this day and age that's a liability. People are going to say we're misogynist.

**Comment by sh4hids** [comment]
1. Clean
2. Elegant
3. Simple
4. Fast
5. Pleasant

**Comment by madmaniak** [comment]
@foxbunny quite exaggerated interpretation. According to games terminology:

2.1. Don't be lame.

**Comment by iamtirado** [comment]
2.1 — The Fastest and Funnest Web App Language

**Comment by iamtirado** [comment]
2.1 — The Funnest Language for the Fastest Web Apps

**Comment by iamtirado** [comment]
2.1 — A simple web language with a warp drive.
![warp drive](https://i.imgur.com/RuJYgPu.gif)

**Comment by foxbunny** [comment]
> Unfairly fast web apps, like child's play.
> Play like a child. Deliver like a pro.

I don't like the "unfarily" in the first one, but I also don't like that the second one does not mention "fast", which is one of the main selling points of Imba. Take "speed" out of Imba, and it's yet another framework that does things differently for the sake of developer ergonomics. I think "fast" is a must. But at the same time, it's for pro work, not just child's play.

**Comment by allain** [comment]
Enjoy delivering fast apps again. Have fun.

**Comment by russoturisto** [comment]
1. Fast Performance
2. Capability/Functionality
3. Fun to Use
4. Beautiful Syntax
5. Rebel

**Comment by iamtirado** [comment]
Code like play, Apps
>> Unfairly fast web apps, like child's play. ...

@foxbunny could you add a thumbs up to hose you like, and thumbs down to those you don't.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by somebee** [comment]
I actually think that we should probably focus most on the simplicity / fun, more so than speed. Yes, Imba is ridiculously fast compared to what most people use today, but will be matched or outperformed by things like Glimmer in the future. All of them, Imba included, will still fall in the realm of "fast enough" to be state management agnostic. But the nicest thing is that Imba achieves this level of performance, yet beats everyone else in conciseness and simplicity (imho) :)

**Comment by foxbunny** [comment]
Or you could just say "Fast" now and change it later when you are the slowest of the pack. Just like a certain popular framework we all know. :D

**Comment by foxbunny** [comment]
All joking aside, "Simple", "Fun", and "Rebel" also seems to come up quite often.

**Comment by iamtirado** [comment]
Hey Everyone @somebee @foxbunny @altteo @madmaniak @russoturisto @allain @sh4hids @nemanjaglumac @athif23
Here's a new tagline that is a simplified composite of taglines mentioned:
Imba: The Fun, Fast, Powerful Web lang.

**Comment by foxbunny** [comment]
Left one looks great

**Comment by iamtirado** [comment]
>Left one looks great

@foxbunny right one is example of the icon in the code editor

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by kalicki** [comment]
> > Left one looks great
> 
> @foxbunny right one is example of the icon in the code editor
> 
> Sent with [GitHawk](http://githawk.com)

IMHO... The logo was cool, but in the code editor it got ugly

**Comment by the94air** [comment]
The logo looks great and I need to mention that things mostly can fly with 2 wings at least or maybe 4 wings. You can add another wing behind the current one. I guess that way, it might be even more cooler.

**Comment by foxbunny** [comment]
> 2 wings at least or maybe 4 wings.

Imba can fly with just one wing. Can you believe that!

**Comment by foxbunny** [comment]
> in the code editor it got ugly

It did not. It's just that it's not quite obvious what the "logo" is on that picture.

**Comment by madmaniak** [comment]
Based on one logo which Imba had for a short period of time.

**Comment by madmaniak** [comment]
Variation based on one logo which Imba had for a short period of time.

![imba-logos](https://user-images.githubusercontent.com/2125923/48967725-5d7fb580-efe5-11e8-972f-940899aeeda9.png)

**Comment by foxbunny** [comment]
Not a big fan of the typeface, but the logo is very sharp. Nice one.

**Comment by madmaniak** [comment]
Yes, typeface is kind of placeholder :smile:. Variation on top of existing logo:

![imbal svg](https://user-images.githubusercontent.com/2125923/48967894-b94b3e00-efe7-11e8-919c-ab0d8ed41f2a.png)

**Comment by athif23** [comment]
Imba word comes from a gaming term, so I think it would be good if the logo has that game feeling in it. 😄 🎮

**Comment by iamtirado** [comment]
Mock up (Proposal) for new brand.
![imba-mockup](https://user-images.githubusercontent.com/13579055/48981594-4ba51d80-f08c-11e8-973f-a30c3f6cd27e.jpg)

Will add description later.

**Comment by the94air** [comment]
@foxbunny :p
